snuba/migrations/operations.py

```diff
@@ -82,6 +84,11 @@ def execute(self) -> None:
     def format_sql(self) -> str:
         raise NotImplementedError
 
+    def __eq__(self, other: Any) -> bool:
```

Comment:
I needed __eq__ on operations for my tests, this seems like a reasonable implementation